doc/source/getting-started.rst

```diff
@@ -0,0 +1,7 @@
+.. _getting-started:
+
+===============
+Getting Started
+===============
+
+.. mdinclude:: ../../README.md
```

Comment:
I gave the README a quick skim - it mentions a list of features this project provides & includes a list of features that mentions listing orchestrations.  It might be easier to say "list users resources, such as datasets" and avoid using the word orchestration

Reply:
Good point.  I should have looked over the readme more carefully.